rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,222 @@
+
+##### Shared Example
+
+Rancher requires environment volumes to already exist prior to creating a service consuming it. Any installed volume driver can be used. We will use the `local` driver because it ships with the Docker daemon.
+
+[Create the volume]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#create-a-volume).
+
+```
+$ rancher volume create --driver local redis-data-external
+```
+
+Define the volume with `external: true` modifier to indicate the volume's lifecycle is managed externally (environment scope). Save the `docker-compose.yml` file below to a directory.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - redis-data-external:/data
+volumes:
+  redis-data-external:
+    driver: local
+    external: true
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+[List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) to verify that `redis-data` volume is `active`.
+
+
+> **Note:** Scaling a service up or down mounts or unmounts the same shared volume.
+
+##### Per-Container Example
+
+In environment scope, creating a unique volume for each container of a service is not supported. Consider using [Stack Scope]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#stack-scope{{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) volumes, or if absolutely necessary, [V1 Compose]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#v1-compose).
+
+#### Stack Scope
+
+A stack-scoped volume is owned and managed by the stack in which it is defined. The primary benefit is the volume lifecycle is automatically managed by Rancher as part of the stack lifecycle. Generic, self-documenting volume names may be defined without colliding with volume names in other stacks.
+
+##### Shared Example
+
+Save the `docker-compose.yml` file below to a directory. Notice that no modifiers are added to the volume; stack scope is default behavior.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - redis-data:/data
+volumes:
+  redis-data:
+    driver: local
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+[List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) to verify that a `redis-data` volume exists. The volume name will be in the form `<stack-name>_<volume-name>_<hash>`.
+
+> **Note:** No auth restrictions are placed on stack-scoped volumes. These volumes could theoretically be mounted by other stacks within the environment if the hash was known.
+
+##### Per-Container Example
```

Comment:
We need to drop references to per_container.
We are leaving it undocumented until we verify all its bugs are fixed. You probably need to drop this entire section.